### DISCUSSION TITLE
Scheduler on Windows

### DISCUSSION THREAD

**Discussion by ButtaJones** [Taxel/PlexTraktSync#733] [Q&A] [answered]
Currently running Windows 10. I originally set it up to run at 4AM daily. Where do I go to change this settings?

**Comment by simonc56** [comment, marked as answer]
You can change it by editing the `Plex Trakt Sync` task in the Windows [Task Scheduler.](https://en.wikipedia.org/wiki/Windows_Task_Scheduler)
